shared/oae/css/oae.base.css

```diff
@@ -578,6 +627,13 @@ ul.as-selections li.as-selection-item.as-ghost-item.as-ghost-selected a.as-close
     display: block;
 }
 
+/* No results message */
+.oae-no-empty-text li.as-message {
```

Comment:
As this class doesn't really apply to anything other than the autosuggest, it would be worth making the class name less generic (e.g. `.oae-as-noresults`)